test/Spago/Build/Polyrepo.purs

```diff
@@ -387,7 +388,7 @@ spec = Spec.describe "polyrepo" do
       , spagoYaml: Init.defaultConfig' $ PackageOnly
           { name: mkPackageName "case-four-package-a"
           , dependencies: [ "console", "effect", "prelude" ]
-          , test: Just { moduleMain: "Subpackage.SameName.Test.Main", strict: Nothing, censorTestWarnings: Nothing }
+          , test: Just { moduleMain: "Subpackage.SameName.Test.Main", strict: Nothing, censorTestWarnings: Nothing, pedanticPackages: Nothing, dependencies: Nothing }
```

Comment:
I wonder if it's worth defining a default config for these polyrepo tests that we override?

Reply:
I definitely need to clean up some of these tests, but I'm not sure if that should be done in this PR.

Reply:
Opened an issue here: https://github.com/purescript/spago/issues/1079